artifactory/commands/golang/archive.go

```diff
@@ -152,7 +149,26 @@ func archiveProject(writer io.Writer, dir, mod, version string) error {
 		return err
 	}
 
-	return Create(writer, m, files)
+	return gozip.Create(writer, m, files)
+}
+
+func getAbsolutePaths(exclusionPatterns []string) ([]string, error) {
```

Comment:
Let's add a unit test for this function. Note that `exclusionPatterns` can include wildcards, so let's add patterns with wildcards to the tests table as well.